### PR TITLE
Add weekly update 2016-03-14

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/115850854) :+1:

**Comment by benjamingr** [comment]
Not sure if on-topic: It could be cool to mention new collaborators were added and mention that the project is always looking for more people to get involved.

**Comment by evanlucas** [comment]
I agree with @benjamingr. I think mentioning the onboarding of the new collaborators would be a good thing

**Comment by Trott** [comment]
@benjamingr @evanlucas Would one of you be willing to write up a draft paragraph for that?

**Comment by benjamingr** [comment]
I started writing something but my English isn't that great, here is the onboarding issue https://github.com/nodejs/node/issues/5064 . 

Pinging @nodejs/inclusivity and @nodejs/evangelism for someone with better prose skills than I. It would be cool to state people are encouraged to participate and become collaborators.

**Comment by evanlucas** [comment]
I can try to take a crack at it this afternoon

**Comment by phillipj** [comment]
@evanlucas any progress on this?

**Comment by evanlucas** [comment]
Ah totally slipped my mind and I've got a pretty busy rest of the week. Can someone else give it a go?

File: locale/en/blog/weekly-updates/weekly-update.2016-03-14.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### Community Updates
+
+* [Presentation: Top Node.js Metrics to Watch](http://blog.sematext.com/2016/02/26/top-node-js-metrics-to-watch/)
+
+If you have spotted or written something about Node.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [NodeConf Adventure 2016](https://ti.to/nodeconf/adventure-2016), "First batch of NodeConf Adventure tickets are up!", June 9th–12th, 2016 - Walker Creek Ranch, Marin, CA, USA
+* [NationJS Node Day Conference](http://nationjs.com/), TICKETS ARE AVAILABLE NOW, March 11, 2016 - Washington, DC
```
Comment (fhemberger) [inline locale/en/blog/weekly-updates/weekly-update.2016-03-14.md:41]: This event already took place.

**Comment by fhemberger** [comment]
@JungMinu Posting looks good to me, as @nodejs/evangelism seems to be busy, I'm merging this now. Regarding a paragraph about the new members, this can be added to the next news as well.

**Comment by fhemberger** [comment]
Merged through d8dfeec.

**Pull request closed without merging**